debian/rules

```diff
@@ -9,8 +9,6 @@ override_dh_auto_configure:
 override_dh_systemd_start:
 	dh_systemd_start --no-start
 
-# Enable all units here, except swap, since dh-systemd doesn't get the
```

Comment:
If we remove the override, `dh_systemd_enable` will try to enable all units. The only units not listed there, `eos-extra-resize.service` and `var-endless\x2dextra.mount`, do not have a `[Install]` section, so I believe `dh_systemd_enable` will do the right thing and not enable them (since they can't be enabled by `systemctl`). I'll send an updated version removing the section altogether.